src/linux/Packaging.Linux/build.sh

```diff
@@ -160,7 +160,7 @@ mkdir -p "$DEBROOT/DEBIAN" "$INSTALL_TO" "$LINK_TO" || exit 1
 
 # make the debian control file
 cat >"$DEBROOT/DEBIAN/control" <<EOF
-Package: gcmcore
```

Comment:
I wonder if this change might break `dpkg` somehow? I guess not? And since we don't publish to apt-get feeds anymore this is also not impacted?

Reply:
I think I got a little ahead of myself on this. I've reverted this change for now and will revisit when we decide to rename the exes/assemblies.